features/old/cms/pages.feature

```diff
@@ -39,7 +39,7 @@ Feature: CMS Pages
         | Path           | /potato       |
         | Content type   | text/css      |
         | Handler        | markdown      |
-        | Tag list       | potato salad |
+        | Tag list       | potato, salad |
```

Comment:
So before it was space separated list?
If so then this is a major change, we should verify that we do not break customer workflow

Reply:
Before it showed space separated, but when you saved, it was saves as a single tag. So customer saves as "tag1, tag2, tag3", then on page reload it appeared as "tag1 tag2 tag3" and if saved again would save as a single tag "tag1 tag2 tag3". Basically now we avoid customer messing up tags by mistake and document how to split tags properly.

Reply:
see https://issues.redhat.com/browse/THREESCALE-7898